src/Forms/GridField/GridField.php

```diff
@@ -552,6 +580,13 @@ public function FieldHolder($properties = [])
             if ($item instanceof GridFieldPaginator) {
                 $total = $item->getTotalItems();
             }
+            if ($sudoModeTransformation) {
+                // Modify the GridFieldViewButton on any GridFields so that it doesn't suffix the view URL with 'view'
+                // This allows us to gracefully reload a form in readonly mode when sudo mode is activated
+                if ($item instanceof GridFieldViewButton) {
```

Comment:
I'd normally ask for this to be combined into a single conditional statement but I think we're both tired of this review cycle, and the change is only a very minor code style change, so will leave as is.